runelite-client/src/main/java/net/runelite/client/plugins/zoom/ZoomPlugin.java

```diff
@@ -67,6 +77,30 @@ public void onScriptEvent(ScriptCallbackEvent event)
 
 		int[] intStack = client.getIntStack();
 		int intStackSize = client.getIntStackSize();
+
+		switch (event.getEventName())
+		{
+			case "panelZoomStart":
+				panelZoom = true;
+				break;
+			case "panelZoomEnd":
+				panelZoom = false;
+				break;
+			case "shouldZoom":
+			{
+				if (!zoomConfig.requireControlDown() || panelZoom || controlDown)
```

Comment:
This logic can be simplified. The ints being compared are by default not equal, so you don't need to explicitly set them to be not equal when this condition is false.